scans/auth/auth_plan_tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Script for testing Automation Framework features
+
+RES=0
+
+mkdir -p /zap/wrk/output
+
+echo "Authentication integration tests"
+echo
+
+cd /zap/wrk/.github/test_resources/plans_and_scripts/
+
+export JIGSAW_USER="guest"
+export JIGSAW_PWORD="guest"
```

Comment:
These shouldnt be here?

Reply:
I trusted the iTests, I have no idea why they're there either :grinning: